Task_7/app/view/RegistrationView.js

```diff
@@ -5,404 +5,387 @@ class RegistrationView extends View {
+              </button>
+            </div>
+          </form>
         </div>
       </div>
-        `;
+    </div>
+      `;
   }
 
   addEvents() {
+    const background = document.querySelector('.background');
     const linkLogin = document.querySelector('.link_enter_login');
     const linkContinue = document.querySelector('.link_enter_continue');
+    const loginLabel = document.querySelector('.label_login');
+    const loginInput = document.querySelector('#login');
+    const nameLabel = document.querySelector('.label_name');
+    const nameInput = document.querySelector('#name');
+    const passwordLabel = document.querySelector('.label_password');
+    const passwordInput = document.querySelector('#password');
+    const passwordRepeatLabel = document.querySelector(
+      '.label_repeat-password'
+    );
+    const passwordRepeatInput = document.querySelector('#repeat_password');
+    const errorMessageLogin = document.querySelector('.input__message-login');
+    const errorMessageName = document.querySelector('.input__message-name');
+    const errorMessagePassword = document.querySelector(
+      '.input__message-password'
+    );
+    const errorMessageRepeatPassword = document.querySelector(
+      '.input__message-repeat-password'
+    );
+    const buttonSubmit = document.querySelector('.button__form_reg');
+    const form = document.querySelector('form');
+    const passwordIcon = document.querySelector('.password');
+    const passwordRepeat = document.querySelector('.password-repeat');
+
     linkLogin.addEventListener('click', () => {
       taskController.createLoginView();
-      background.remove();
     });
 
     linkContinue.addEventListener('click', () => {
       taskController.createMainView();
       background.remove();
     });
 
-    const loginLabel = document.querySelector('.label_login');
-    const loginInput = document.querySelector('#login');
-    const errorMessageLogin = document.querySelector('.input__message-login');
-
-    loginInput.addEventListener('keyup', () => {
-      errorMessageLogin.innerHTML = '';
+    loginInput.addEventListener('input', () => {
+      Helper.removeSpaceInput(loginInput);
+      Helper.addInputMessage([errorMessageLogin, ERRORS.emptyMessage]);
       if (
         !(loginInput.value.length && Helper.checkLatinLetters(loginInput.value))
       ) {
-        Helper.changeStatusInput(
-          loginLabel,
-          loginInput,
-          'label_validate',
-          'input_validate',
-          'label_error',
-          'input_error'
+        Helper.addInputView(
+          [loginLabel, HTML_CLASS.input.labelError],
+          [loginInput, HTML_CLASS.input.inputError]
+        );
+        Helper.removeInputView(
+          [loginLabel, HTML_CLASS.input.labelValidate],
+          [loginInput, HTML_CLASS.input.inputValidate]
         );
+        Helper.addInputMessage([
+          errorMessageLogin,
+          ERRORS.loginValidateMessage,
+        ]);
       } else {
```

Comment:
don't use block else
add return in the end of if block (after  Helper.removeInputView)